addons/base_vat/models/res_partner.py

```diff
@@ -206,6 +206,13 @@ def _construct_constraint_msg(self, country_code):
 
     __check_vat_ch_re = re.compile(r'E([0-9]{9}|-[0-9]{3}\.[0-9]{3}\.[0-9]{3})(MWST|TVA|IVA)$')
 
+    def check_vat_xi(self, vat):
+        """ Temporary Nothern Ireland VAT validation following Brexit
+        As of January 1st 2021, companies in Northern Ireland have a
+        new VAT number starting with XI
+        TODO: remove when stdnum is updated in supported distro"""
```

Comment:
Can you mention the version of stdnum needed to remove this patch? (1.16)
Maybe also the commit linked in stdnum https://github.com/arthurdejong/python-stdnum/commit/53f13b4af087250e43f8841c01906e46d5687b44 in you commit message

Reply:
Maybe https://github.com/arthurdejong/python-stdnum/commit/b93d69581f35aa18e7fdd52b3f7fdf06770215e3 actually

Reply:
ok, done